meson.build

```
@@ -35,6 +35,7 @@ tasks_deps = [
 ]
 
 if libecal_dep.version().version_compare('<=3.39')
+    add_project_arguments('--define=EDS_3_39', language: 'vala')
     tasks_deps += meson.get_compiler('vala').find_library('libecal-2.0-fixes', dirs: meson.current_source_dir() / 'vapi')
 endif
```

We usually do the reverse for readability
```suggestion
if libecal_dep.version().version_compare('>=3.39.2')
    add_project_arguments('--define=HAS_EDS_3_40', language: 'vala')
else
    tasks_deps += meson.get_compiler('vala').find_library('libecal-2.0-fixes', dirs: meson.current_source_dir() / 'vapi')
endif
```

---

wow, that was quick :)

makes sense, will implement as advised!